.github/workflows/publish-package-to-npmjs.yml

```diff
@@ -29,8 +19,6 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
-          cache: "npm"
-          cache-dependency-path: ${{ env.WORKING_DIR }}/package-lock.json
```

Comment:
Why were these lines removed?

Reply:
This build is running before package publish now, thus, there will not be a cache for it